nix/ci-config.nix

```diff
@@ -67,6 +67,17 @@ in
       );
   };
   config = {
+    # NOTE(Emily, 22 Mar 2023): Here we provide sensible defaults for working with liqwid-nix. 
+    # These can be overwritten by the user, though. In the future, we may want some out-of-the-
+    # -box implementation for enabling this somehow.
+    flake = { ... }: {
+      # Added in: 2.7.2.
+      config.herculesCI = {
+        ciSystems = [ "x86_64-linux" ];
+        onPush.default.outputs = self.checks.x86_64-linux;
+        onPush.required.outputs = self.checks.x86_64-linux.required;
+      };
+    };
```

Comment:
Note: for liqwid-offchain we want some of our (longer running) tests to only run on some branches, but we can override this in the flake.nix there.

Reply:
Yep. This is all just defaults. See also: https://docs.hercules-ci.com/hercules-ci-agent/evaluation.html